src/styles/About.css

```diff
@@ -37,17 +51,18 @@
   float: right;
   width: 80px;
   padding: 10px;
-  border-radius: 15px;
+  border-radius: 80px;
   box-shadow: 5px 5px 5px grey;
   margin-right: 40px;
-
+  background-color: white;
 }
 
 .RightBlurbImages {
   float: left;
   width: 80px;
   padding: 10px;
+  border-radius: 80px;
   box-shadow: 5px 5px 5px grey;
-  border-radius: 15px;
   margin-left: 40px;
+  background-color: white;
 }
```

Comment:
About.css: Good styling of elements, spacing between the cliparts and blurbs, and presentation of each clipart.